### ISSUE TITLE
Server stop responding on HTTP request to HTTPS listener

### ISSUE THREAD

**Issue by antonkomarev** [mpyw/php-hyper-builtin-server#9]
I've launched 2 listeners:

```sh
hyper-run -S localhost:8000 -s localhost:4000 -c ~/cert.pem -t ./public
```

`https://localhost:4000` working as expected until you will try to go to the `http://localhost:4000`. After then https listener stops responding on requests displaying permanent loading and white screen.

**Comment by antonkomarev** [comment]
Could you tell me where to dig? I'll try to fix this issue then.

**Comment by antonkomarev** [comment]
Resolved in #10
